config/application.rb

```diff
@@ -23,6 +23,6 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
 
-    config.exceptions_app = routes
+    config.exceptions_app = self.routes
```

Comment:
why was this change needed?

Reply:
It wasn't, I was just trying to get the custom error pages to show instead of the debug error pages

Reply:
Thanks for explaining, I've not come across it before which is why I was asking 😄